pkg/ratelimit/limiter.go

```diff
@@ -20,8 +20,19 @@ import (
 	"golang.org/x/time/rate"
 )
 
+// DimensionConfig is the limit dimension config of one label
+type DimensionConfig struct {
```

Comment:
Could you please try to remove `DimensionConfig` and use `LabelLimitters` instead of `Limitter`?
Since we  already store the config in `RateLimiter` and `concurrencyLimiter` and they are all theadsafe
```
type limiter struct {
	qps         *RateLimiter
	concurrency *concurrencyLimiter
}

type LabelLimitters struct {
	limiters sync.Map // label->limiter
	labelAllowList map[string]struct{}
}
```